modules/govuk_jenkins/manifests/job/backdrop_transactions_explorer_collector.pp

```diff
@@ -0,0 +1,32 @@
+# == Class: govuk_jenkins::job::backdrop_transactions_explorer_collector
+#
+# Backdrop collector for the Transactions Explorer Google Spreadsheet
+#
+class govuk_jenkins::job::backdrop_transactions_explorer_collector (
+  $google_credentials = undef,
+  $google_spreadsheet_key = undef,
+  $google_spreadsheet_worksheet = undef,
+  $backdrop_token = undef,
+) {
+
+  govuk::app::envvar {
+    "${title}-GOOGLE_CREDENTIALS":
```

Comment:
Ah, sorry, I think I misunderstood slightly. You'd probably do this explicitly in the Jenkins shell part of the template rather than creating env vars to load in ie:
`export GOOGLE_CREDENTIALS=<%= @google_credentials %>`